lib/chess/auth/user.ex

```diff
@@ -11,6 +11,7 @@ defmodule Chess.Auth.User do
     field :username, :string
 
     has_many :games, Chess.Store.Game
+    has_many :other_games, Chess.Store.Game, foreign_key: :opponent_id
```

Comment:
This doesn't seem like a helpful name to me. Is there another descriptive name, like, `opponent_games`?